### ISSUE TITLE
I do not have the Midbase ID

### ISSUE THREAD

**Issue by GerhardABSA** [CameronABSA/Activity-Tracker-AT-#62]
How do I find an item where I have some info but not the midbase id.
Will I be able to do a search on name registration number and SDS Id and not only the midbase number in order to find the entity.

How would I search say with a director or a contact person some times if I would like to do some workflow. 
Will we be able to link the CSM to the entity so we have the connected CSM from that point onward until we asssign a different CSM etc.

How would we be able to link multiple relationships to an entity.

**Comment by CameronABSA** [comment]
Good idea, What I am thinking is the CSM has a profile and has the User Category "CSM" already, when they log workflow, the AT tool will recognize they are a CSM and use their user _ID to populate the CSM field for this entity. Does this suffice? 

The multiple field search I shall build in :+1:
